tests/test_api/test_list_jobs.py

```diff
@@ -188,7 +188,7 @@ def test_bad_date_formats(client):
         '2020-13-01T00:00:00Z',
         '01-JAN-2020',
         '01/01/2020',
-        '2020-01-01' '2020-01-01T00:00Z',
+        '2020-01-012020-01-01T00:00Z',
```

Comment:
I expect these were intended as two different test cases, i.e.
```suggestion
        '2020-01-01',
        '2020-01-01T00:00Z',
```

Reply:
Here's what it looked like before ruff cleanup started; missing comma was a bug
![image](https://github.com/user-attachments/assets/64a73cb1-013a-4fa0-a790-baace188aadf)